mopidy/mpd/protocol/stored_playlists.py

```diff
@@ -157,7 +166,15 @@ def playlistclear(context, name):
 
         Clears the playlist ``NAME.m3u``.
     """
-    raise exceptions.MpdNotImplemented  # TODO
+    uri = context.lookup_playlist_uri_from_name(name)
+    playlist = uri is not None and context.core.playlists.lookup(uri).get()
+    if not playlist:
```

Comment:
Fair enough, I would add comment about this and ideally also a test case which does more or less what you just pasted as a quick sanity check that we don't break this in the future.